harvester/records/record.py

```diff
@@ -228,6 +232,78 @@ def is_deleted(self) -> bool:
             return True
         return False
 
+    def get_controlled_dct_format_s_term(self, value: str | None) -> str | None:
```

Comment:
I'm assuming you've done some analysis of the original vs the controlled term, not looking for hard numbers but what was your general sense of the accuracy of these crosswalks?

Reply:
Good question.  It's accurate, particularly for MIT data.  The field this pulls from in FGDC (ISO is not a problem) returns maybe 10-20 unique values, but this mapping here captures the various forms and maps to controlled terms for us.

I should have kept the examples/analysis, but I think they're gone to the wind, but the mapping basically covers the values seen for our data sans a couple of really strange little stragglers.